apps/api-journeys/src/modules/button/button.spec.ts

```diff
@@ -24,13 +24,14 @@ describe('ButtonModule', () => {
     dbMock.journey.findUnique.mockResolvedValue({
       id: journeyId,
       title: 'published',
-      published: true,
       locale: 'en-US',
       themeMode: ThemeMode.light,
       themeName: ThemeName.base,
       description: null,
       primaryImageBlockId: null,
-      slug: 'published-slug'
+      slug: 'published-slug',
+      publishedAt: new Date(),
```

Comment:
```suggestion
      publishedAt: '2021-11-15T03:42:22.322Z',
```
Use a hard coded value in these all the component tests that aren't Journey, they are here just as a placeholder - we don't need to properly test `publishedAt` and `createdAt` here.

Reply:
Can't use a hardcoded string as publishedAt needs to be a date object. As discussed we will hard code a date object in.